proto/src/determined/api/v1/experiment.proto

```diff
@@ -57,6 +57,8 @@ message GetExperimentsRequest {
     SORT_BY_USER = 10;
     // Returns experiments sorted by name.
     SORT_BY_NAME = 11;
+    // Returns experiments sorted by originating model
```

Comment:
Proto is really finicky, you need to have this comment end with a period to show up in the generated TypeScript

Reply:
Oh I think it is showing up in this case? 🤔  Anyway it's better to have it

Reply:
Ok I made change